UI/Pages/Contactus.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@page
+@model ContactusModel
+@{
+	ViewData["Title"] = "Contactus";
```

Comment:
The page's filename should be ContactUs (capital U) and the title should be "Contact us" two separate words